tyssue/core/history.py

```diff
@@ -377,7 +377,7 @@ def record(self, to_record=None, time_stamp=None, sheet=None):
                 if "segment" in df.columns:
                     kwargs["min_itemsize"] = {"segment": 8}
                 with pd.HDFStore(self.hf5file, "a") as store:
-                    if element in store and self.time in store.select(element, columns=["time"])["time"].unique():
+                    if element in store and len(store.select(element, where=f"time == {self.time}")['time'].unique())>0:
```

Comment:
maybe `store.select(element, where=f"time == {self.time}").shape[0] > 0` is shorter? We don't need to call `unique` I think